CHANGELOG.md

```diff
@@ -14,6 +14,8 @@ All notable changes for each version of this project will be documented in this
     ```typescript
     public pinningConfiguration: IPinningConfig = { columns: ColumnPinningPosition.End };
     ```
+- `IgxCombo`:
+    - Added `autoFocusSearch` input that allows to manipulate the combo's opening behavior. When the property is `true` (by default), the combo's search input will be focused after the `onOpened` event is fired (current behavior). When set to `false`, the focus will go to the combo items' list container. Suppressing the input focus can be used to prevent a keyboard showing up on mobile devices when opening the combo.
```

Comment:
The description with the event and current behavior seem a bit too much in detail. Perhaps a shorter version?
```suggestion
    - Added `autoFocusSearch` input that allows to manipulate the combo's opening behavior. When the property is `true` (by default), the combo's search input is focused on open. When set to `false`, the focus goes to the combo items container, which can be used to prevent the software keyboard from activating on mobile devices when opening the combo.
```